Cython/Compiler/Tests/TestParseTreeTransforms.py

```diff
@@ -240,33 +240,31 @@ def test_debug_info(self):
             funcnames = ('codefile.spam', 'codefile.ham', 'codefile.eggs',
                          'codefile.closure', 'codefile.inner')
             required_xml_attrs = 'name', 'cname', 'qualified_name'
-            assert all(f in xml_funcs for f in funcnames)
+            self.assertTrue(all(f in xml_funcs for f in funcnames))
```

Comment:
If this fails, it will be unclear which is missing.
```suggestion
            self.assertFalse([f for f in funcnames if f not in xml_funcs])
```

Reply:
Your code will work fine, but I think that if we want this to be more obvious we can use sets:
```python
set(funcnames) - set(xml_funcs)
```

(As well as use it instead of `issubset`)

Reply:
[resolved](https://github.com/cython/cython/pull/4340/commits/6541aa8a97f39065b7629b432ccc1abd1a862bc6)